ql/src/java/org/apache/hadoop/hive/ql/io/RecordReaderWrapper.java

```diff
@@ -69,7 +70,14 @@ static RecordReader create(InputFormat inputFormat, HiveInputFormat.HiveInputSpl
       JobConf jobConf, Reporter reporter) throws IOException {
     int headerCount = Utilities.getHeaderCount(tableDesc);
     int footerCount = Utilities.getFooterCount(tableDesc, jobConf);
-    RecordReader innerReader = inputFormat.getRecordReader(split.getInputSplit(), jobConf, reporter);
+
+    RecordReader innerReader = null;
+    try {
+     innerReader = inputFormat.getRecordReader(split.getInputSplit(), jobConf, reporter);
+    } catch (InterruptedIOException iioe) {
+      // If reading from the underlying record reader is interrupted, return a no-op record reader
```

Comment:
Answer is here & this does fixes a couple of test so I picked it:
https://github.com/apache/hive/pull/1742/files#r674896581

Reply:
Does this log line work:
```
      LOG.info("Interrupted while getting the input reader for {}", split.getInputSplit());
```
For 2nd I suppose it can get interrupted on any abort, not very sure, do you have any suggestions